include/SDL_revision.h

```diff
@@ -1,2 +1,6 @@
+#ifdef SDL_VENDOR_INFO
```

Comment:
This doesn't make sense in this context. This would be the default header if someone copied all the headers from an unconfigured source archive, and there's nothing to set SDL_VENDOR_INFO. If someone defined that in their application, that still wouldn't affect the build unless they were also building SDL sources.

I'm thinking this change can be dropped.

Reply:
Isn't the unconfigured version the one that's used to build with MSVC or Xcode, if you do a `git clone` and build it in the obvious way? I thought we'd want to let people do a `-DSDL_VENDOR_INFO="for ioquake3"` or whatever in their project configuration, even if there's nothing nicer available.

(But I don't use those IDEs, so I don't really know how they work.)

Reply:
Fair enough, never mind. :)